adoc/Private-Registry-Installation.adoc

```diff
@@ -361,40 +360,35 @@ persistence:
   persistentVolumeClaim:
     registry:
       storageClass: ""
-      accessMode:
-      size:
-    jobservice:
-      storageClass: ""
-      accessMode:
+      accessMode: ReadWriteMany
       size:
     database:
       storageClass: ""
-      accessMode:
+      accessMode: ReadWriteOnce
```

Comment:
Isn't this (and following `ReadWriteOnce` entries) actually default value? So why should we reconfigure it?

Reply:
Good point, let me fix that.